dev/doc/release-process.md

```diff
@@ -52,13 +53,14 @@
 - [ ] Push the new tag with `git push upstream VX.X... --dry-run` (remove the `--dry-run` and redo if everything looks OK).
 - [ ] Set `is_a_released_version` to `false` (if you forget about it, you'll be reminded by the test-suite failing whenever you try to backport a PR with a changelog entry).
 - [ ] Close the milestone.
-- [ ] Send an e-mail on coqdev + coq-club + the Coq announcement category on Discourse (coqdev@inria.fr + coq-club@inria.fr + coq+announcements@discoursemail.com) announcing that the tag has been set so that package managers can start preparing package updates (including a `coq-bignums` compatible version), after which library authors can safely start preparing compatible releases.
-- [ ] In particular, ensure that someone is working on providing an opam package (either in the main [ocaml/opam-repository](https://github.com/ocaml/opam-repository) for standard releases or in the `core-dev` category of the [coq/opam-coq-archive](https://github.com/coq/opam-coq-archive) for preview releases.
-- [ ] Make sure to cc `@ejgallego` to ensure that a release for [coq-serapi](https://github.com/ejgallego/coq-serapi) is available in `opam-repository` (typically, using `dune-release tag` and `dune-release`).
-- [ ] Make sure to cc `@proux01` to ensure that a `coq-bignums` opam package is available in [`extra-dev`](https://github.com/coq/opam-coq-archive/tree/master/extra-dev) or [`released`](https://github.com/coq/opam-coq-archive/tree/master/released), respectively.
-- [ ] Make sure to cc `@erikmd` to ensure that the necessary configuration is ready to release the Docker images in [`coqorg/coq`](https://hub.docker.com/r/coqorg/coq) (gathering `coq`, `coq-bignums`, and `coq-serapi` opam packages).
 - [ ] Publish a release on GitHub with the PDF version of the reference manual attached. The PDF can be recovered from the artifacts of the `doc:refman-pdf` job from continuous integration. Also attach a `tar.gz` archive of the sources (to ensure a stable hash, you can copy the archive autogenerated by github when the release is published).
+- [ ] Ensure that someone is working on providing an opam package (either in the main [ocaml/opam-repository](https://github.com/ocaml/opam-repository) for standard releases or in the `core-dev` category of the [coq/opam-coq-archive](https://github.com/coq/opam-coq-archive) for preview releases.
 - [ ] If pinged by opam package providers in pull requests to [ocaml/opam-repository](https://github.com/ocaml/opam-repository), transfer any changes to opam packages required by opam-repository CI (such as missing dependencies) to the corresponding package definitions in the Coq repository.
+- [ ] Cc `@ejgallego` to ensure that a release for [coq-serapi](https://github.com/ejgallego/coq-serapi) is available in `opam-repository` (typically, using `dune-release tag` and `dune-release`).
```

Comment:
AFAIU, SerAPI is no longer maintained and thus no longer included in Docker-Coq images, therefore, this line can be removed. (Cf. https://github.com/coq-community/manifesto/issues/160)

Reply:
@erikmd do you confirm?

Reply:
@erikmd done that